IndexedDB/remove-listener-in-handler.htm

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>removeEventListener in a handler doesn't interfere with future transactions</title>
+<link rel="author" title="quasicomputational" href="https://github.com/quasicomputational">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="support.js"></script>
+<script>
+const t = async_test();
+
+const STORE = "store";
+
+let count = 0;
+
+const go = () => {
+    const dbReq = indexedDB.open("db", 1);
+    dbReq.addEventListener("upgradeneeded", (ev) => {
+        ev.target.result.createObjectStore(STORE, { autoIncrement: true });
+    });
+    dbReq.addEventListener("success", (ev) => {
+        const db = ev.target.result
+        const tx = db.transaction([STORE], "readwrite");
```

Comment:
nit: With a single store in scope, the `[]` are not needed. Harmless, but it's extra noise.